Sources/GDSCommon/UI/UIKit/SharedViews/Patterns/Popover/PopoverTableViewController.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+import UIKit
+
+/// View Controller for the `PopoverTableView` storyboard XIB
+/// This screen includes the following views:
+///   - `tableView` (type: `UITableView`)
+/// This screen provides a list of options for the user, in a popover view.
+public final class PopoverTableViewController: UIViewController {
+    
+    /// tableView label: ``UITableView``
+    @IBOutlet private var tableView: UITableView! {
+        didSet {
+            tableView.accessibilityIdentifier = "table-view"
+        }
+    }
+    
+    var items: [PopoverItemViewModel] = []
+    
+    /// Initialiser for the `PopoverTableView` view controller.
+    /// Requires a single parameter.
+    /// - Parameter items: `[PopoverItemViewModel]`
+    public init(items: [PopoverItemViewModel]) {
+        self.items = items
+        super.init(nibName: "PopoverTableView", bundle: .module)
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    public override func viewDidLayoutSubviews() {
+        self.preferredContentSize = CGSize(width: self.view.frame.width, height: tableView.contentSize.height)
```

Comment:
could we turn this in the private computed property that returns this GCSize? would avoid it being duplicated